### ISSUE TITLE
App crashes immediately after lunching on Android

### ISSUE THREAD

**Issue by heliomaster** [kivy/kivy#3839]
There seem to be a problem running apps on Android. It works fine I'm when running it on Mac OS but crashes immediately after the source files have been extracted on Android. I'm using buildozer to build it and what I can see there are no problems when building the app.
This word for word the same issue as Alecktos on issue #3644 with a different message however.
Here is the log:

```
I/ActivityManager(  631): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=org.test.njeapp/org.renpy.android.PythonActivity} from uid 10011 on display 0
D/LauncherLog( 1441): Launcher:: startActivity: pkg: org.test.njeapp clz: org.renpy.android.PythonActivity
I/ActivityManager(  631): Launched start pid= [ -1]
I/art     ( 6638): Late-enabling -Xcheck:jni
I/ActivityManager(  631): Start proc org.test.njeapp:python for activity org.test.njeapp/org.renpy.android.PythonActivity: pid=6638 uid=10138 gids={50138, 9997, 1028, 1015} abi=armeabi
V/Launcher( 1441): [onWindowFocusChanged]: false
D/Launcher.Utilities( 1441): isPhone= false
D/Launcher.Utilities( 1441): isPhone= false
D/houdini ( 6638): [6638] Initialize library(version: 5.0.5_x.47634 RELEASE)... successfully.
D/Launcher.Utilities( 1441): isPhone= false
D/saved_wallpaper_scroll( 1441): save cur wallpaper offset x: 0.5, State: APPS_CUSTOMIZE
D/WindowManager(  631): adjustConfigurationLw, config:{0 1.0 ?mcc?mnc ?locale ?layoutDir sw600dp w600dp h951dp 160dpi lrg port ?uimode ?night finger -keyb/v/h -nav/v} mLidState:-1 mHasDockFeature:false mHasKeyboardFeature:false mHasHallSensorFeature:false config.hardKeyboardHidden:2
V/ActivityManager(  631): Display changed displayId=0
I/ActivityManager(  631): Launched start pid= [ 6638]
V/GoogleTagManager( 1441): Dispatch call queued. Dispatch will run once initialization is complete.
V/Monotype( 6638): SetAppTypeFace- try to flip, app = org.test.njeapp
V/Monotype( 6638):     Typeface getFontPathFlipFont - systemFont = default#default
V/Monotype( 6638): SetAppTypeFace- try to flip, app = org.test.njeapp
V/Monotype( 6638):     Typeface getFontPathFlipFont - systemFont = default#default
W/ResourceType( 6638): No package identifier when getting value for resource number 0x00000000
V/python  ( 6638): metadata fullscreen is0
I/SDLSurface( 6638): Surface will NOT be transparent
D/OpenGLRenderer( 6638): Render dirty regions requested: false
W/ResourceType( 6638): No package identifier when getting value for resource number 0x00000000
D/Atlas   ( 6638): Validating map...
D/houdini ( 6638): [6657] Added shared library /data/app/org.test.njeapp-2/lib/arm/libsdl.so for ClassLoader by Native Bridge.
D/houdini ( 6638): [6657] Added shared library /data/app/org.test.njeapp-2/lib/arm/libsdl_image.so for ClassLoader by Native Bridge.
E/        ( 6638): release-version: 1.04 
E/        ( 6638):  build time: '17:00:54' - 'Jul  5 2015'
I/OpenGLRenderer( 6638): Initialized EGL, version 1.4
D/OpenGLRenderer( 6638): Enabling debug mode 0
D/houdini ( 6638): [6657] Added shared library /data/app/org.test.njeapp-2/lib/arm/libsdl_ttf.so for ClassLoader by Native Bridge.
D/houdini ( 6638): [6657] Added shared library /data/app/org.test.njeapp-2/lib/arm/libsdl_mixer.so for ClassLoader by Native Bridge.
D/houdini ( 6638): [6657] Load library /data/app/org.test.njeapp-2/lib/arm/libpython2.7.so failed.
E/art     ( 6638): dlopen("/data/app/org.test.njeapp-2/lib/arm/libpython2.7.so", RTLD_LAZY) failed: dlopen failed: "/data/app/org.test.njeapp-2/lib/arm/libpython2.7.so" has unexpected e_machine: 40
--------- beginning of crash
E/AndroidRuntime( 6638): FATAL EXCEPTION: Thread-947
E/AndroidRuntime( 6638): Process: org.test.njeapp:python, PID: 6638
E/AndroidRuntime( 6638): java.lang.UnsatisfiedLinkError: dlopen failed: "/data/app/org.test.njeapp-2/lib/arm/libpython2.7.so" has unexpected e_machine: 40
E/AndroidRuntime( 6638): 	at java.lang.Runtime.loadLibrary(Runtime.java:371)
E/AndroidRuntime( 6638): 	at java.lang.System.loadLibrary(System.java:989)
E/AndroidRuntime( 6638): 	at org.renpy.android.PythonActivity.run(PythonActivity.java:254)
E/AndroidRuntime( 6638): 	at java.lang.Thread.run(Thread.java:818)
D/ActivityManager(  631): Prepare to broadcast crash information.
D/ActivityManager(  631): Broadcast crash info message!
W/ActivityManager(  631):   Force finishing activity org.test.njeapp/org.renpy.android.PythonActivity
I/WindowManager(  631): Screenshot max retries 4 of Token{2fa8011d ActivityRecord{6bab5f4 u0 org.test.njeapp/org.renpy.android.PythonActivity t268 f}} appWin=Window{1853389 u0 SurfaceView} drawState=1
E/ActivityManager(  631): Invalid thumbnail dimensions: 360x360
D/Atlas   (  631): Validating map...
I/SDLSurface( 6638): Choose egl configuration
I/SDLSurface( 6638): Try to use graphics config R8G8B8A8S8
W/SDLSurface( 6638): Create egl surface
I/Choreographer( 6638): Skipped 30 frames!  The application may be doing too much work on its main thread.
I/OpenGLRenderer(  631): Initialized EGL, version 1.4
I/python  ( 6638): presplash-fit is null
I/python  ( 6638): presplash (fit=null) mx=0,585938,my=0,928711
W/InputMethodManagerService(  631): Focus gain on non-focused client com.android.internal.view.IInputMethodClient$Stub$Proxy@3f6c2ef2 (uid=10138 pid=6638)
D/WindowManager(  631): adjustConfigurationLw, config:{0 1.0 ?mcc?mnc ?locale ?layoutDir sw600dp w600dp h951dp 160dpi lrg port ?uimode ?night finger -keyb/v/h -nav/v} mLidState:-1 mHasDockFeature:false mHasKeyboardFeature:false mHasHallSensorFeature:false config.hardKeyboardHidden:2
V/Launcher( 1441): [onResume] called
I/Launcher( 1441): Time spent in onResume: 6ms
I/ActivityManager(  631): Killing 6512:android.process.acore/u0a21 (adj 15): empty #9
W/SDLSurface( 6638): onDestroy() called
W/libprocessgroup(  631): failed to open /acct/uid_10021/pid_6512/cgroup.procs: No such file or directory
D/SDLSurface( 6638): onDestroy() stop requested, wait for an event from the app
```
I have encompassed i hope the process incriminated.

**Comment by heliomaster** [comment]
Hi @kived,
I've posted a better part of the log and opened a new issue. Could you direct me to at least how to start fixing this issue. For the moment I use a  linux virtual machine and manage to create an apk this way but i really would like to have buildozer work on the mac :-)

**Comment by dessant** [comment]
Duplicate of https://github.com/kivy/python-for-android/issues/653.
